doc/rtd/reference/datasources/aeza.rst

```diff
@@ -0,0 +1,36 @@
+.. _datasource_aeza:
+
+Aeza
+****
+
+The `Aeza`_ datasource consumes the content served from Aeza's metadata
+service. This metadata service serves information about the running VPS
+via http at ``77.221.156.49``.
+
+Configuration
+=============
+
+Aeza's datasource can be configured as follows: ::
+
+  datasource:
+    Aeza:
+      metadata_url: "http://77.221.156.49/v1/cloudinit/{id}/meta-data"
+      userdata_url: "http://77.221.156.49/v1/cloudinit/{id}/user-data"
+      vendordata_url: "http://77.221.156.49/v1/cloudinit/{id}/vendor-data"
```

Comment:
One design question from a configuration standpoint: do we really want to have 3 separate configuration keys for meta-data/vendor-data and user-data? There are all routes under `http://77.221.156.49/v1/cloudinit/{id}/`. Do you expect some deployments would require very unique separate URLs that don't share the same prefix?

If not may we just use `metadata_url: "http://77.221.156.49/v1/cloudinit/{id}/"`?

Reply:
Fixed in 4e8b32dfd7ce0751ca0781a6894600e2e70f1a2a